decisiontree/models.py

```diff
@@ -198,6 +199,27 @@ def __str__(self):
         return u"%s : %s --> %s" % (self.current_state, self.answer, self.next_state)
 
 
+class SessionQuerySet(models.query.QuerySet):
+
+    def open(self):
+        return self.exclude(Q(state=None) | Q(canceled=True))
+
+    def complete(self):
+        return self.filter(Q(state=None) | Q(canceled=True))
+
+
+class SessionManager(models.Manager):
```

Comment:
Sounds good! Since Django 1.8 isn't far out :four_leaf_clover: it's probably best to just move forward.